lib/rodauth/features/change_login.rb

```diff
@@ -51,9 +52,8 @@ module Rodauth
             end
 
             after_change_login
-            set_notice_flash change_login_notice_flash
-            redirect change_login_redirect
           end
+          change_login_response
```

Comment:
This moves the code from inside the transaction block to outside.  Not a big deal in this case, but we should probably attempt to avoid unnecessary changes when refactoring.

Reply:
I figured this was an overlook from the initial implementation, given you probably don't want to enforce the redirection from within the transaction (and none of the other implementations do?). Was I wrong in this assessment?

Reply:
No.  I think the change makes sense.  However, it's probably better as a separate commit.